architecture/additional_concepts/authorization.adoc

```diff
@@ -880,6 +880,40 @@ in their SCC set.  This allows cluster administrators to run pods as any
 user by without specifying a `RunAsUser` on the pod's `SecurityContext`.  The
 administrator may still specify a `RunAsUser` if they wish.
 
+[[rbac-to-scc]]
+==== Role-Based Access to SCCs ====
+Starting with version 3.11, you can specify Security Context Constrains as a
+resource that is handled by roles or cluster roles. This allows you to
+scope the access to your SCCs to a certain project. Assigning users/groups
+directly to an SCC retains cluster-wide scope.
+
+To include access to SCCs for your role, you need to specify the following rule:
+.Role-Based Access to SCCs
+[source,yaml]
+----
+rules:
+  apiGroups:
+  - security.openshift.io
```

Comment:
Each of these things need to have call outs with explanations, even if it something as simple as "this is the group that SCC is in"